### PR TITLE
Translate zh file Useful-Websites-to-Learn-How-to-Code-Quickly.md  into en

### PR DESCRIPTION
**Pull request by MaxwellHawk**
Simple translation stuff. syntax errors need to be corrected.

### CODE REVIEW

File: en/Useful-Websites-to-Learn-How-to-Code-Quickly.md [hunk 1]
```diff
@@ -0,0 +1,95 @@
+#  Website recommendations for developers to learn programming 
+
+* en：[9 Useful Websites to Learn How to Code Quickly](http://www.queness.com/post/10709/8-useful-websites-to-learn-how-to-code-quickly)
+* zh translation:  [wangguo](http://wangguo.iteye.com/)
+
+The Internet is a rapidly evolving field, where technologies constantly being updated, like from HTML to HTML5, from CSS to CSS3, from JavaScript to JavaScript frameworks, etc. If you can't keep up with the speed, you will be out of game. Therefore, a quick grasp of a language or a technology can be very beneficial.
```
Comment (scarydemon2) [inline en/Useful-Websites-to-Learn-How-to-Code-Quickly.md]: you will be out of game   ------->the game over

File: en/Useful-Websites-to-Learn-How-to-Code-Quickly.md [hunk 2]
```diff
@@ -0,0 +1,95 @@
...
+
+The site contains a tutorial that lets you start your MongoDB learning journey with some commands. 
+
+![](images/9-useful-mongodb.jpg)
+
+## [学堂在线](https://next.xuetangx.com/)
+
+Technology: WeChat small program development, algorithm This is an online education website supported by Tsinghua University and other institutions.
+
+It includes courses such as recommended algorithms, WeChat mini-program production, etc.
```
Comment (Aupateli) [inline en/Useful-Websites-to-Learn-How-to-Code-Quickly.md]: It includes courses such as recommendation algorithms, WeChat mini-program development, etc.

File: en/Useful-Websites-to-Learn-How-to-Code-Quickly.md [hunk 3]
```diff
@@ -0,0 +1,95 @@
...
+
+Languages: Java, Python, Go, PHP and other mainstream development languages This is the official website of MongoDB. MongoDB is a scalable, high-performance, open-source NoSQL database written in the C? language. For java, Python, Go, PHP and other mainstream development languages. 
+
+The site contains a tutorial that lets you start your MongoDB learning journey with some commands. 
+
+![](images/9-useful-mongodb.jpg)
+
+## [学堂在线](https://next.xuetangx.com/)
+
+Technology: WeChat small program development, algorithm This is an online education website supported by Tsinghua University and other institutions.
```
Comment (Aupateli) [inline en/Useful-Websites-to-Learn-How-to-Code-Quickly.md]: Technology: WeChat mini-program development

File: en/Useful-Websites-to-Learn-How-to-Code-Quickly.md [hunk 4]
```diff
@@ -0,0 +1,95 @@
...
+
+Language: PHP 
+
+This is not an interactive way to learn, but it has a good written tutorial that contains a wealth of examples and guides to teach you the basics of PHP and MySQL. 
+
+![](images/9-useful-phpKnowHow.jpg)
+
+## [MongoDB](http://www.mongodb.org/)
+
+Languages: Java, Python, Go, PHP and other mainstream development languages This is the official website of MongoDB. MongoDB is a scalable, high-performance, open-source NoSQL database written in the C? language. For java, Python, Go, PHP and other mainstream development languages.
```
Comment (Aupateli) [inline en/Useful-Websites-to-Learn-How-to-Code-Quickly.md]: Languages: Java, Python, Go, PHP and other mainstream development languages.
This is the official website of MongoDB. MongoDB is a scalable, high-performance, open-source NoSQL database written in C++. MongoDB applies to java, Python, Go, PHP and other mainstream development languages.

File: en/Useful-Websites-to-Learn-How-to-Code-Quickly.md [hunk 5]
```diff
@@ -0,0 +1,95 @@
...
+
+Learn about Java and Python by solving sample problems and online coding practices. 
+
+![](images/9-useful-codingBat.jpg)
+
+## [PHP Know How](http://www.phpknowhow.com/)
+
+Language: PHP 
+
+This is not an interactive way to learn, but it has a good written tutorial that contains a wealth of examples and guides to teach you the basics of PHP and MySQL.
```
Comment (Aupateli) [inline en/Useful-Websites-to-Learn-How-to-Code-Quickly.md]: This is not an interactive way to learn, but it has a well written tutorial that contains a wealth of examples and guides to teach you the basics of PHP and MySQL.

File: en/Useful-Websites-to-Learn-How-to-Code-Quickly.md [hunk 6]
```diff
@@ -0,0 +1,95 @@
...
+
+Each lesson has a video and detailed articles to make sure you understand them.  
+
+![LifeHacker](images/9-useful-LifeHacker.jpg)
+
+##  [CodeCademy](http://www.codecademy.com/)
+
+Language: JavaScript 
+
+This is an interactive form of instruction that gives you a bit of a grip on JavaScript.
```
Comment (Aupateli) [inline en/Useful-Websites-to-Learn-How-to-Code-Quickly.md]: This is an instruction with interactive form

File: en/Useful-Websites-to-Learn-How-to-Code-Quickly.md [hunk 7]
```diff
@@ -0,0 +1,95 @@
...
+The Internet is a rapidly evolving field, where technologies constantly being updated, like from HTML to HTML5, from CSS to CSS3, from JavaScript to JavaScript frameworks, etc. If you can't keep up with the speed, you will be out of game. Therefore, a quick grasp of a language or a technology can be very beneficial. 
+
+This article provides you 9 practical online tutorials, some in interactive format, and some in comprehensive guides and visualizations to help you quickly master a language. They share a common purpose, that is, make your learning more fun, and make it easier for you to master it.
+
+##  [LifeHacker Learn to code](http://www.lifehacker.com.au/2011/02/learn-to-code-the-full-beginners-guide/)
+
+Language: JavaScript 
+4 lessons plus best practices/resources to teach you how to program using Javascript. 
+
+Each lesson has a video and detailed articles to make sure you understand them.
```
Comment (Aupateli) [inline en/Useful-Websites-to-Learn-How-to-Code-Quickly.md]: Each lesson has a video and detailed articles to make sure that you understand them.

File: en/Useful-Websites-to-Learn-How-to-Code-Quickly.md [hunk 8]
```diff
@@ -0,0 +1,95 @@
+#  Website recommendations for developers to learn programming
```
Comment (Aupateli) [inline en/Useful-Websites-to-Learn-How-to-Code-Quickly.md]: #  Website recommendations for developers to learn programming quickly

**Pull request merged**